dataplane/linux/ipip_mgr.go

```diff
@@ -195,6 +201,7 @@ func (d *ipipManager) OnUpdate(msg interface{}) {
 	switch msg := msg.(type) {
 	case *proto.HostMetadataUpdate:
 		log.WithField("hostanme", msg.Hostname).Debug("Host update/create")
+		// @shaun: Should this be `msg.Ipv4Addr + "/32"` since we are not type 'hash:net'?
```

Comment:
The ipsets dataplane layer canonicalises the IP using `ParseCIDROrIP()` so shouldn't be a problem.